src/Sylius/Behat/Page/Admin/Promotion/UpdatePageInterface.php

```diff
@@ -42,4 +42,8 @@ public function setEndsAt(\DateTimeInterface $dateTime): void;
     public function hasStartsAt(\DateTimeInterface $dateTime): bool;
 
     public function hasEndsAt(\DateTimeInterface $dateTime): bool;
+
+    public function isCouponManagementAvailable(): bool;
+
+    public function manageCoupons(): void;
```

Comment:
I'd move this function above `hasStartsAt` to group them logically, but it can be done in a separate PR :)